tools/build/setup_macos_android.sh

```diff
@@ -81,12 +87,13 @@ function main() {
   declare -r android_home="${ANDROID_HOME:-$HOME/Library/Android/sdk}"
   install_android_tools "${android_home}"
 
-  install_gradle
-  gradle --version
+  declare -r gradle_home="${HOME}/opt/gradle"
```

Comment:
Is this standard? I've never seen this. Is another directory more appropriate, like $HOME/Library?

Reply:
Yeah,` /opt` is a thing, `$HOME/opt` is not. Lets use `/opt` as the instructions recommend.
```suggestion
  declare -r gradle_home="/opt/gradle"
```

Reply:
Ah, you will need sudo for that. It seems the right place is `/usr/local/opt/gradle`